mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import time
+
+
+def test_check(checkType):
+    """
+    Support test-check API
+    """
+    test_check_responses = {}
+    test_check_responses["agent.cpu"] = [
+        {
+            "timestamp": int(time.time() * 100),
```

Comment:
Nothing in mimic should use the real `time.time` - this function needs to get passed an `IReactorTime` so that the current synthetic time used by the rest of mimic matches it.